src/components/Faq.content.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+export const faqQuestionsAnswers = [
+	{
+		question: 'How much storage do I need?',
+		answer: (
+			<p>
+				A little bit of money can go a long way in data storage. For $1 USD you can purchase storage for
+				approximately 1000 documents, 55 photos, or a 30 second video. Files of the same type (docs, jpegs, or
+				mp3) will vary in sizes, but some rules of thumb are: 1 document 0.31 MB (320KB); 1 HD Photo 2.5 MB; 3.5
+				minute Song 3.5 MB; 1 minute HD video 100 MB.
+			</p>
+		)
+	},
+	{
+		question: 'How are fees calculated?',
+		answer: (
+			<p>
+				The file size is used to determine the base fee to upload data to the network. The price of the base fee
+				is set by the Arweave Network code itself. The Price Calculator includes this base fee and adds the
+				ArDrive Community Tip percentage on top. Learn more about{' '}
+				<a href="https://ardrive.atlassian.net/wiki/spaces/help/pages/86376465">ArDrive Fees</a>.
```

Comment:
We'll add the following sentence just below to disambiguate data size representations:
```
Note: All file sizes are represented using binary units of measurement (i.e. 1 MB = 1024 KB).
```